mk/config.mk

```diff
@@ -641,7 +641,7 @@ endif
 #   is a totally standard shared object, and should be used to link against.
 #   The '.ta' file is a signed version of the '.so' and should be installed
 #   in the same way as TAs so that they can be found at runtime.
-CFG_ULIBS_SHARED ?= n
+CFG_ULIBS_SHARED ?= y
```

Comment:
unrelated change. Prefer discard it.

Reply:
Sorry. Got confused while doing something else.